main.js

```diff
@@ -9,11 +9,28 @@ const { ipcMain } = require('electron')
 let mainWindow
 
 function createWindow() {
-    mainWindow = new BrowserWindow({width: 700, height: 600,
-        frame: false,"node-integration": true})
+    welcomeWindow = new BrowserWindow({
+        width: 700, height: 600,
+        frame: false, "node-integration": true
+    })
```

Comment:
All statements should end with a semicolon.

Reply:
fixed.